.github/workflows/ci.yml

```diff
@@ -378,6 +378,31 @@ jobs:
             --target x86_64-unknown-linux-gnu \
             --verbose
 
+  vortex-duckdb-test:
+    name: "Rust tests (vortex-duckdb)"
+    timeout-minutes: 120
+    runs-on:
+      - runs-on=${{ github.run_id }}
+      - family=c6id.8xlarge
```

Comment:
Picked a beefy machine such that the build tasks matches the duration of others, not introducing any CI wait time.

Reply:
I would stick to the regular machines, unless the perf difference is really big? I am always suspicious of this one of setup for ci

Reply:
We have to build DuckDB, so this build task would take significantly longer than others.

Reply:
Giving it a try just now.